x-pack/plugins/security_solution/public/resolver/store/data/reducer.test.ts

```diff
@@ -48,5 +90,127 @@ describe('Resolver Data Middleware', () => {
     it('should indicate there are additional children', () => {
       expect(selectors.hasMoreChildren(store.getState())).toBe(true);
     });
+    it('should have the correct related events', () => {
+      const selectedEventsByEntityId = selectors.relatedEventsByEntityId(store.getState());
+      const selectedEventsForFirstChildNode = selectedEventsByEntityId.get(firstChildNodeInTree.id)!
+        .events;
+
+      expect(selectedEventsForFirstChildNode).toBe(firstChildNodeInTree.relatedEvents);
+    });
+    it('should indicate the correct related event count for each category', () => {
+      const selectedRelatedInfo = selectors.relatedEventInfoByEntityId(store.getState());
+      const displayCountsForCategory = selectedRelatedInfo.get(firstChildNodeInTree.id)
+        ?.getNumberActuallyDisplayedForCategory!;
+      for (const typeCounted of Object.keys(statsForFirstChild)) {
+        expect(`${typeCounted}:${displayCountsForCategory(typeCounted)}`).toBe(
+          `${typeCounted}:${statsForFirstChild[typeCounted]}`
+        );
+      }
+    });
+    it('should not indicate the limit has been exceeded when the actual count >= stats count', () => {
```

Comment:
can you explain the reasoning behind this? `'should not indicate the limit has been exceeded when the actual count >= stats count'`

Reply:
Also, maybe:
```
'should not indicate the limit has been exceeded when the actual related event counts for a category >= stats count'
```
(assuming that I'm understanding this correctly.)

Reply:
https://github.com/elastic/kibana/pull/70097/commits/9671aa0adb5702c520a7c2a3cdca7df3c188637a